templates/hound.yml

```diff
@@ -0,0 +1,11 @@
+# See https://houndci.com/configuration for configuration details.
+coffeescript:
+  enabled: true
+haml:
+  enabled: true
+javascript:
+  enabled: false
```

Comment:
I was thinking thoughtbot's guides recommend CS https://github.com/thoughtbot/guides/tree/master/best-practices#javascript. Suspenders provides defaults for what a thoughtbot project would be most of the time and if you want something different, you can be explicit. Should we flip it now that ES6 is becoming more common?